pkg/syncer/cnsoperator/controller/cnsregistervolume/cnsregistervolume_controller.go

```diff
@@ -86,6 +89,9 @@ func Add(mgr manager.Manager, clusterFlavor cnstypes.CnsClusterFlavor,
 		log.Debug("Not initializing the CnsRegisterVolume Controller as its a non-WCP CSI deployment")
 		return nil
 	}
+	workloadDomainIsolationEnabled = commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx,
+		common.WorkloadDomainIsolation)
+
```

Comment:
do we need to enable this controller for stretched supervisor case ? it looks disabled now 

 else {
				if len(clusterComputeResourceMoIds) > 1 {
					log.Infof("Not initializing the CnsRegisterVolume Controller as stretched supervisor is detected.")
					return nil
				}
			}